src/executor/query.rs

```diff
@@ -108,21 +110,21 @@ macro_rules! try_getable_all {
                     QueryResultRow::SqlxMySql(row) => {
                         use sqlx::Row;
                         row.try_get::<Option<$type>, _>(column.as_str())
-                            .map_err(|e| TryGetError::DbErr(crate::sqlx_error_to_query_err(e)))
+                            .map_err(|e| TryGetError::DbErr(DbBackend::MySql.map_query_err(e)))
```

Comment:
We don't need to map the error into specific DB error at `src/executor/query.rs` because the query has been executed. We are parsing the result here. We can keep it as `crate::sqlx_error_to_query_err(e)`.